__fixtures__/output1/helpers.ts

```diff
@@ -221,22 +221,22 @@ const fromJSON = (object: any): Timestamp => {
 };
 
 const timestampFromJSON = (object: any): Timestamp => {
-    return {
-        seconds: isSet(object.seconds) ? Long.fromValue(object.seconds) : Long.ZERO,
-        nanos: isSet(object.nanos) ? Number(object.nanos) : 0,
-    };
+  return {
+    seconds: isSet(object.seconds) ? Long.fromValue(object.seconds) : Long.ZERO,
+    nanos: isSet(object.nanos) ? Number(object.nanos) : 0,
+  };
 }
-
+  
 export function fromJsonTimestamp(o: any): Timestamp {
-    if (o instanceof Date) {
-        return toTimestamp(o);
-    } else if (typeof o === "string") {
-        return toTimestamp(new Date(o));
-    } else {
-        return timestampFromJSON(o);
-    }
+  if (o instanceof Date) {
+    return toTimestamp(o);
+  } else if (typeof o === "string") {
+    return toTimestamp(new Date(o));
+  } else {
+    return timestampFromJSON(o);
+  }
 }
-
+  
 function numberToLong(number: number) {
```

Comment:
does `numberToLong` always get created?

Reply:
yes, because this function's with the helper file, which will be created as a whole in create-helper generator. It's behavior changes when switching between bigint and Long.